### PR TITLE
Rename package to proper format.

### PR DESCRIPTION
**Pull request by jqn**

### CODE REVIEW

File: app/src/main/java/org/codefordenver/encorelink/LoginActivity.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        login_button.setOnClickListener(
+                new View.OnClickListener() {
+                    @Override
+                    public void onClick(View view) {
+                        Log.d(TAG, "Login");
+
+                        if (!validate()) {
+                            onLoginFailed();
+                            return;
+                        }
```
Comment (gea-07) [inline app/src/main/java/org/codefordenver/encorelink/LoginActivity.java]: Validation is in the wrong spot. Need to grab user name and password first before calling validate and onLoginFailed.

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <Button
+        android:id="@+id/button_login"
+        android:layout_width="match_parent"
+        android:layout_height="40dp"
+        android:layout_margin="10dp"
+        android:text="Login"
+        android:textColor="#FFFFFF"
+        android:background="@color/colorPrimaryDark"
+        />
+
```
Comment (gea-07) [inline app/src/main/res/layout/activity_login.xml]: Need to let people register for new accounts with a register button.

File: app/src/main/java/org/codefordenver/encorelink/LoginActivity.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+                            new Handler().postDelayed(
+                                    new Runnable() {
+                                        @Override
+                                        public void run() {
+                                            onLoginSuccess();
+                                            progressDialog.dismiss();
+                                        }
+                                    }, 3000
+                            );
+                        }
```
Comment (gea-07) [inline app/src/main/java/org/codefordenver/encorelink/LoginActivity.java]: You need an else statement to handle invalid credentials and login failure does not dismiss authentication dialog when entering wrong credentials

**Comment by jqn** [comment]
Closing this in a effort to clean up repo. Please feel free to reopen if relevant.

**Pull request closed without merging**